onedal/svm/svm.py

```diff
@@ -165,6 +165,9 @@ def _get_sample_weight(self, X, y, sample_weight):
 
     def _get_onedal_params(self, data):
         max_iter = 10000 if self.max_iter == -1 else self.max_iter
+        # TODO: remove this workaround
+        # when oneDAL SVM starts support of 'n_iterations' result
+        self.n_iter_ = 1 if max_iter < 1 else max_iter
```

Comment:
If we don't have this feature lets do fallback on sklearnex side

Reply:
it's not feature or parameter, but output
`n_iter_` is for number of computed svm iterations